src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: #060b35;
+  padding: 32px 16px 16px;
+
+ &:not(:nth-child(4n)) {
+  margin-right: $card-gap;
+ }
+
+ &:nth-child(-n + 4) {
+  margin-bottom: 46px;
+ }
+
+  &__imac-img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: #616070;
+    font-size: 10px;
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    @include flex-base;
+    margin-bottom: 24px;
+  }
+
+  &__rewiev {
+    font-size: 10px;
+  }
+
+  &__price {
+    @include flex-base;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: #616070;
+    line-height: 18px;
+  }
+
+  &__price-sum {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: inline-block;
+    background-color: #00acdc;
+    width: 100%;
```

Comment:
items inside a flex container will automatically have the display property set to block. and the block elements have 100% width by default.
for that reason, display and width are unnecessary here